docs/user_guide/assessment/perform_fairness_assessment.rst

```diff
@@ -21,14 +21,23 @@ We next examine these four steps in more detail.
 Identify types of harms
 -----------------------
 
-See :ref:`types_of_harms` for a guide to types of fairness-related harms. 
-For example, in a system for screening job applications, qualified candidates 
-that are automatically rejected experience an allocation harm. In a 
-speech-to-text transcription system, disparities in word error rates for 
-different groups may result in harms due to differences in the quality of service.
+See :ref:`types_of_harms` for a guide to types of fairness-related harms.
+The Fairlearn package is particularly suitable for measuring:
+
+* *Allocation Harms* occur when a system unfairly extends or witholds
+  opportunities, resources, or information.
+  Common (but by no means exhaustive) examples are hiring for jobs, student
+  admissions and loan origination
+
+* *Quality of Service Harms* occur when a system works much better for
+  one group than another.
+  For example, facial recognition and speech-to-text systems may have
+  substantially different performance for different ethnicities
+
 Note that one system can lead to multiple harms, and different types of 
-harms are not mutually exclusive. For more information, review 
-Fairlearn's `2021 SciPy tutorial <https://github.com/fairlearn/talks/blob/main/2021_scipy_tutorial/overview.pdf>`_.
+harms are not mutually exclusive.
```

Comment:
Random note: someone pointed out to me the other day that "harms" apparently is not really a word in English (?) only singular "harm". But given that none of our native English speakers have been bothered by "harms" perhaps that's not true?

Reply:
It didn't have any fingernails-on-blackboard effect on me, although that's not the last word on the matter. As a verb, 'harms' is certainly a word, but we're using it as a noun. I suppose it comes down to whether harm is considered 'continuous' or 'discrete' (those might not be the grammatical terms). I'm thinking of 'a box near some other boxes' vs. 'some sugar.'

Reply:
Ok! It seems that "officially" harm is more like sugar, but we're not the grammar police so as long as people don't mind I'm happy to leave it in - just wanted to check whether I've hurt any English speakers eyes and ears by using harms as a noun ;)

Reply:
I believe it is a noun as well - unless it's just a [gerund](https://en.wikipedia.org/wiki/Gerund). However, I'd better refrain from that particular wiki-wormhole first thing on a Monday morning.